client/db/types.go

```diff
@@ -193,6 +193,12 @@ type MetaMatch struct {
 	Match *order.UserMatch
 }
 
+// SetStatus sets the match status in both the UserMatch and the MatchMetaData.
+func (m *MetaMatch) SetStatus(status order.MatchStatus) {
```

Comment:
Why is this exported now?

Reply:
Used from core/trade.go via `(*matchTracker).SetStatus`, and directly via `(*db.MetaMatch).SetStatus` for the maker/taker cancel match meta data.

Reply:
Derp. I was reading this wrong. This is good.